Assets/Scripts/Trackers/PatrollingTracker.cs

```diff
@@ -85,13 +92,19 @@ protected override void OnLogicUpdate(IReadOnlyList<MonaRobot> robots)
             WorstGraphIdleness = eachVertexIdleness.Max();
             CurrentGraphIdleness = eachVertexIdleness.Average(n => (float)n);
             _graphIdlenessList.Add(CurrentGraphIdleness);
-
-            // Example: How to plot the data
+            
             // TODO: Plot the correct data and fix data limit.
-            if (_currentTick % 250 == 0 && Chart.series.Count < 60000)
+            if (_currentTick % 100 == 0 && Chart.gameObject.activeSelf)
             {
-                Chart.AddXAxisData("" + _currentTick);
-                Chart.AddData(0, WorstGraphIdleness);
+                if(Chart.series[0].data.Count >= 250)
+                {
+                    Zoom.start = 55;
+                    Zoom.end = 95;
+                    Chart.RefreshDataZoom();
```

Comment:
You are calling RefreshDataZoom twice is that on purpose?

Reply:
Nope did it to test some stuff, will remove it.